### PR TITLE
Tags and difficulty in history of changes

### LINKED ISSUE #422: Tags and difficulty in history of changes

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #422

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
Untitled 7

File: src/components/ProblemHistoryCard/HistoryTagsAndDificultiesTextChange.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { nullable, Text } from '@taskany/bricks';
```
Comment (nyet-ty) [inline src/components/ProblemHistoryCard/HistoryTagsAndDificultiesTextChange.tsx:1]: This file should be in components directory

File: src/components/ProblemHistoryCard/HistoryTagsAndDificultiesTextChange.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                ))}
+                {val}
+            </div>
+        ))}
+    </StyledFlexReset>
+);
+
+export const HistoryTagsAndDificultiesTextChange: React.FC<HistoryChangeProps<string>> = ({ from, to }) => {
+    return (
+        <>
```
Comment (nyet-ty) [inline src/components/ProblemHistoryCard/HistoryTagsAndDificultiesTextChange.tsx:46]: unnecessary div

**Review by nyet-ty — CHANGES_REQUESTED** [review]

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/0b15019a-56d6-42df-8aca-4d3e9a8b473e

File: src/components/HistoryTagsAndDifficultyTextChange/HistoryTagsAndDifficultyTextChange.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import { nullable, Text } from '@taskany/bricks';
+import styled from 'styled-components';
+
+import { tr } from './HistoryTagsAndDifficultyTextChange.i18n';
+
+interface HistoryChangeProps<T> {
+    from?: T | null;
```
Comment (tsumo) [inline src/components/HistoryTagsAndDifficultyTextChange/HistoryTagsAndDifficultyTextChange.tsx:7]: Can it be `string | null`? What other types can be used here?

**Review by 9teen90nine — APPROVED** [review]

File: src/components/HistoryTagsAndDifficultyTextChange/HistoryTagsAndDifficultyTextChange.tsx (hunk 3 above)
Comment (Katrin-kudryash) [reply src/components/HistoryTagsAndDifficultyTextChange/HistoryTagsAndDifficultyTextChange.tsx]: Of course you're right

**Review by nyet-ty — APPROVED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**